pkg/ipmi/oem.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// OEM implementation
+package ipmi
+
+import (
+	"github.com/u-root/u-root/pkg/smbios"
+)
+
+func (i *IPMI) RunOEMCommands(si *smbios.Info) error {
```

Comment:
Is this going to contain many IPMI commands? Can each IPMI command be a different function?

Reply:
Yes, each IPMI should be a different function, something like
RunOEMCommands() {
    i.oemSetProcessDimmInfo()
    i.oemSyncBootOrder()
    ...
}

Reply:
Why not directly call oemSetProcessDimmInfo and oemSyncBootOrder from the uinit?

Reply:
The idea was trying to put all IPMI OEM related implementation into pkg/ipmi/oem.c and keep uinit more generic, if putting them in uinit is also fine then I can remove pkg/ipmi/oem.c in this PR.

Reply:
anyway, we can discuss in the following diffs